include/swift/AST/Decl.h

```diff
@@ -4325,15 +4325,28 @@ class AbstractStorageDecl : public ValueDecl {
 
 /// VarDecl - 'var' and 'let' declarations.
 class VarDecl : public AbstractStorageDecl {
+public:
+  enum class Specifier : uint8_t {
+    // For Var Decls
+    
+    Let  = 0,
+    Var  = 1,
+    
+    // For Param Decls
+    
+    None  = Let,
```

Comment:
If they're semantically equivalent then it's probably fine, but a comment would be helpful. Whether using Owned everywhere would make things more clear I am not sure as I haven't looked over all the uses.